### DISCUSSION TITLE
text indicates that red is not supported ?

### DISCUSSION THREAD

**Discussion by linhuanxia** [tailwindlabs/tailwindcss#12335] [Help]
text indicates that red is not supported ?

I used text-[#EB3B2F] and text-[red] unstyled

![image](https://github.com/tailwindlabs/tailwindcss/assets/38238997/eddcf5f6-2b81-4dad-9aa2-5c47068dc7dc)

![image](https://github.com/tailwindlabs/tailwindcss/assets/38238997/f053ef4b-ee9e-4351-92b6-5c923c9a7f01)

**Comment by linhuanxia** [comment]
I used the orange color [#FF8D1A] for styling , but red not !

![image](https://github.com/tailwindlabs/tailwindcss/assets/38238997/f9fb48ed-f297-4e78-948e-aaef62ae61bc)

**Comment by adamwathan** [comment]
Hey! Works here:

https://play.tailwindcss.com/OnARzOcfca

<img width="1501" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/4323180/f1bc015a-e44b-4290-bbbd-3c34e648bc15">

**Reply by linhuanxia** [reply]
![image](https://github.com/tailwindlabs/tailwindcss/assets/38238997/8e570a81-1425-478d-8bd6-d8402dc878c3)
My style text-[red] just doesn't come out, maybe the table is affecting it

**Reply by jack-tunda** [reply]
The color variable was override by something.

**Reply by adydetra** [reply]
@linhuanxia try using `!text-[red]`

**Reply by wongjn** [reply]
> The color variable was override by something.

There is no variable being used here.

> @linhuanxia try using `!text-[red]`

It does not seem this is a case of CSS specificity that changing `text-[red]` to `!text-[red]` would solve. The rule for `.text-7xl` and `.text-[red]` would show up in devtools and then some other rule would be shown to override `.text-[red]` if it *was* a specificity issue. If changing `text-[red]` to `!text-[red]` *does* solve the issue, I would expect it to be some other underlying issue.

@linhuanxia, consider checking:
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).
